### PR TITLE
fix: distributed online covariance samples fix

### PR DESCRIPTION
**Pull request by Alexandr-Solovev**
# Description
Pr contains the fix for nightly job + removing definition of partial compute(comm version)

### CODE REVIEW

**Comment by Alexandr-Solovev** [comment]
Nightly job: https://intel-ci.intel.com/eed705c2-f813-f15f-b29e-a4bf010d0e2e

File: samples/oneapi/dpc/mpi/sources/cov_online_distr_mpi.cpp [hunk 1]
```diff
@@ -49,8 +49,7 @@ void run(sycl::queue& queue) {
     dal::covariance::partial_compute_result<> partial_result;
 
     for (std::int64_t i = 0; i < nBlocks; i++) {
-        partial_result =
-            dal::preview::partial_compute(queue, cov_desc, partial_result, input_blocks[i]);
+        partial_result = dal::partial_compute(queue, cov_desc, partial_result, input_blocks[i]);
     }
     const auto result = dal::preview::finalize_compute(comm, cov_desc, partial_result);
```
[the remark this answers is not in the document]
Comment (Alexandr-Solovev) [reply samples/oneapi/dpc/mpi/sources/cov_online_distr_mpi.cpp:54]: Yes, all of our distributed functions, like infer(comm, args) train(comm, args), compute(comm, args) are still in preview as well as finalize_compute(comm, args).
We decided that for online algorithms we shouldnt have partial_compute(comm, args) just partial_compute(queue, args), so, in this case I call default one single batch partial_compute+distributed finalize compute

[the remark this answers is not in the document]
Comment (Alexandr-Solovev) [reply samples/oneapi/dpc/mpi/sources/cov_online_distr_mpi.cpp:54]: So, here I call `dal::partial_compute(queue,..)` and `dal::preview::finalize_compute(comm, ..)`

[the remark this answers is not in the document]
Comment (ahuber21) [reply samples/oneapi/dpc/mpi/sources/cov_online_distr_mpi.cpp:54]: Thanks for the explanation :)

**Review by ethanglaser — APPROVED** [review]

**Comment by ethanglaser** [comment]
Changes resolve failures - but is non-preview finalize_compute compatible with spmd?

**Comment by Alexandr-Solovev** [comment]
> Changes resolve failures - but is non-preview partial_compute compatible with spmd?

Hi! We use batch(no distributed partial_compute on each node, and merge it only in finalize step) it means that in partial_compute we dont use comm functions

**Pull request merged**